.gitignore

```diff
@@ -1,3 +1,2 @@
 *~
 my-*
-
```

Comment:
This commit is empty and not related to the text, which is already listed in an earlier commit in the same PR.